local/rest_api_gcbm/tests/tiff/belize/LifeZone_moja.tiff.aux.xml

```diff
@@ -0,0 +1,14 @@
+<PAMDataset>
```

Comment:
@ankitaS11 - you can remove this file, it's an artefact of the pre-processing stage.

Reply:
I believe you can remove `bounding_box.tiff` too.

I know you've had trouble running the GCBM on your Arch distro. However I think the `fatal (306)` error described in #136 may be due to an issue with the simulation configuration.

Are you able to run GCBM Belize using the AppImage you created? If not, I'd be happy to test the Docker containers with you. I don't think there's anything that should stop you from running FLINT on your local machine to validate that your API generated config is valid.

Reply:
Thanks, @aornugent! I'll remove these files, I think you are right. It's a good time to try running the simulation through the docker containers locally. I'll give it a try again over the weekend and reach out to you with specific issues I'm aware of.

Reply:
Done! :)